src/main.cpp

```diff
@@ -8,6 +10,20 @@ int main(int argc, char *argv[])
     QCoreApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
     KiwixApp a(argc, argv);
 
+    // format systems language
+    QString defaultLocale = QLocale::system().name(); // e.g. "de_DE"
+    defaultLocale.truncate(defaultLocale.lastIndexOf('_')); // e.g. "de"
+    QLocale::setDefault(defaultLocale);
+    
+    QTranslator qtTranslator;
+    qtTranslator.load("qt_" + QLocale::system().name(),
+                             QLibraryInfo::location(QLibraryInfo::TranslationsPath));
+    a.installTranslator(&qtTranslator);
+
+    QTranslator myappTranslator;
+    myappTranslator.load(":/i18n/kiwix-desktop.qm");
+    a.installTranslator(&myappTranslator);
+
```

Comment:
All those initialization would be better directly in the KiwixApp code.

Reply:
Ok